### ISSUE TITLE
Wrong dependencies names for express.js

### ISSUE THREAD

**Issue by dmitryhryppa** [haxiomic/dts2hx#12]
Hi, 

I've tried to use this awesome library with express.js, and looks like I found a bug. 
Or I did something wrong :) 

Express itself does not have type definitions, so I installed them from `@types/express` and my `package.json` dependencies looks like:
```
	"devDependencies": {
		"@types/express": "^4.17.1",
		"dts2hx": "^0.6.0"
	},
	"dependencies": {
		"express": "^4.17.1"
	}
```

Then after executing `npx dts2hx express` it successfully generates externs, but some `haxelib.json` files of generated externs was created with dependencies like this:
```
	"dependencies": {
		"@types/node": "",
		"@types/express-serve-static-core": "",
		"@types/serve-static": ""
	},
```

So the problem is  `@types/` which is not a valid name for Haxe library, and to get it working I removed `@type/` for all dependencies by hands:
```
	"dependencies": {
		"node": "",
		"express-serve-static-core": "",
		"serve-static": ""
	},
```

P.S. Thank you for this library. It is a big step forward for Haxe ecosystem!

**Comment by haxiomic** [comment]
Thanks for testing @dmitryhryppa! I see what's going on (there's two forms of module ids in the converter and I'm using the wrong one for dependencies). Will fix tomorrow :)

**Comment by haxiomic** [comment]
Fixed in 0.7.1 :)
